### PR TITLE
Incorporate more EUV models 

### PR DESCRIPTION
**Pull request by aaronjridley**
# Description

Addresses #85 

We have incorporated the 59 wavelength model from GITM as well as the 22 wavelength model from Solomon's paper. These are included as CSV files with the right cross sections.

## Type of change

- New feature - this adds a couple more EUV models for the user to choose from.

# How Has This Been Tested?

A test has been added in tests/euv.  This tests the code in a bunch of different configurations, since EUVAC is available for all of the different wavelength files.

## Test configuration

* Operating system: Max OS 13.4; also on Linux Ubuntu
* Compiler, version number: gcc version 12.3 and 11.2
* Any details about your local setup that are relevant - didn't turn on Fortran or netCDF

# Checklist:

- [N/A] Make sure you are merging into the ``develop`` (not ``master``) branch
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (see the EUV section of the overleaf document)
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [N/A] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: src/calc_ion_temperature.cpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ void Ions::calc_ion_temperature(Neutrals neutrals, Grid grid,
 
   std::string function = "Ions::calc_ion_temperature";
   static int iFunction = -1;
-  report.enter(function, iFunction);
+  // report.enter(function, iFunction);
```
Comment (aburrell) [inline src/calc_ion_temperature.cpp:44]: Remove code instead of commenting it out, here and elsewhere.

File: tests/euv/run_test.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
```
Comment (aburrell) [inline tests/euv/run_test.sh:1]: Can you add a line to run this test to the yaml in `.github/workflow`?

**Review by aburrell — CHANGES_REQUESTED** [review]

File: src/calc_chemical_sources.cpp [hunk 3]
```diff
@@ -55,19 +55,17 @@ void Chemistry::calc_chemical_sources(Neutrals &neutrals,
...
 
     change3d.fill(rate);
-
+    
     // check for type of temperature dependence and calculate
     if (reactions[iReaction].type > 0) {
       // Determined which temperature to use in equation:
-      // use Ti by default
-      arma_cube temp = ions.temperature_scgc;
+      // use Tn by default
```
[the remark this answers is not in the document]
Comment (aaronjridley) [reply src/calc_chemical_sources.cpp:61]: TI is more common

File: tests/euv/run_test.sh (hunk 2 above)
Comment (aaronjridley) [reply tests/euv/run_test.sh:1]: I don't know how to do this and I am unsure if we should have all tests run for all pull requests.  We should chat about how to do this.

File: src/calc_euv.cpp [hunk 4]
```diff
@@ -37,7 +37,13 @@ int calc_euv(Planets planet,
     // Chapman integrals for EUV energy deposition:
     neutrals.calc_chapman(grid, report);
 
-    iErr = euv.euvac(time, indices, report);
+    if (input.get_euv_model() == "euvac")
+      iErr = euv.euvac(time, indices, report);
+    if (input.get_euv_model() == "neuvac")
+      iErr = euv.neuvac(time, indices, report);
+    if (input.get_euv_model() == "hfg")
+      iErr = euv.solomon_hfg(time, indices, report);
```
[the remark this answers is not in the document]
Comment (aaronjridley) [reply src/calc_euv.cpp]: I changed this

**Comment by aburrell** [comment]
In the future, remember not to merge until you get an approval on the pull request.

**Pull request merged**